torch/nn/modules/_functions.py

```diff
@@ -46,9 +46,8 @@ def forward(self, input, weight, bias, running_mean, running_var, eps, momentum,
             count_all.view(-1).long().tolist()
         )
 
-        self.save_for_backward(input, weight, mean, invstd)
+        self.save_for_backward(input, weight, mean, invstd, count_all)
```

Comment:
nit: the permissions of this file have changed

Reply:
Good catch. Will undo the permission change.